source/index.txt

```diff
@@ -1,118 +1,113 @@
-.. _index:
-
 ================
-MongoDB C Driver
+{+driver-long+}
 ================
 
-.. facet::
-   :name: programming_language
-   :values: c
-
-.. facet::
-   :name: genre
-   :values: reference
-
-.. meta:: 
-   :keywords: code example, get started, sample app
-
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: twocols
-
 .. toctree::
    :titlesonly:
    :maxdepth: 1
 
-   /libbson
-   /libmongoc
+   libbson API Documentation <{+api-libbson+}>
+   libmongoc API Documentation <{+api-libmongoc+}>
 
 Introduction
 ------------
 
-Welcome to the documentation site for the official MongoDB C driver.
-You can add the driver to your application to work with MongoDB in C.
-Download the required libraries, ``libmongoc`` and ``libbson``, by following
-the :ref:`obtaining_libraries` instructions or set up a runnable project by
-following our tutorial.
-
-- :ref:`Tutorial <mongoc_tutorial>`
-
-- :ref:`Usage Guide <libmongoc>`
+Welcome to the documentation site for the {+driver-long+}, also known as **libmongoc**,
```

Comment:
Q: are we just documenting libmongoc?

Reply:
Answered offline – summary is we plan to include `libbson` info in the proper pages in the new standardized ToC (Installation instructions, BSON guide)